src/components/timestampcontrol/TimestampControlDirective.js

```diff
@@ -0,0 +1,34 @@
+(function() {
+  goog.provide('ga_timestamp_control_directive');
+
+  var module = angular.module('ga_timestamp_control_directive', []);
+
+  module.directive('gaTimestampControl', function($rootScope) {
+    return {
+      restrict: 'A',
+      scope: {
+        map: '=gaTimestampControlMap'
+      },
+      templateUrl: 'components/timestampcontrol/partials/timestampcontrol.html',
+      link: function(scope, element) {
```

Comment:
General for this directive: why not use a scope variable for the html content (which would be put inside the partial) instead of using element.find().html() methods? I think it would be more angular like.